src/main/java/org/reaktivity/nukleus/tls/internal/stream/ServerStreamFactory.java

```diff
@@ -198,7 +203,8 @@ private MessageConsumer newAcceptStream(
         {
             final TlsRouteExFW routeEx = route.extension().get(tlsRouteExRO.get()::wrap);
             final String store = routeEx.store().asString();
-            final SSLContext sslContext = lookupContext.apply(store);
+            final StoreInfo storeInfo = lookupContext.apply(store);
+            final SSLContext sslContext = storeInfo == null ? null : storeInfo.context;
```

Comment:
Seems like this one is really `lookupContext`, no need to let `StoreInfo` into `ServerStreamFactory`.

Reply:
It uses supportsClientAuth below. So kept as StoreInfo